gateway-service/src/main/java/org/zowe/apiml/gateway/filters/PageRedirectionFilterFactory.java

```diff
@@ -174,7 +177,7 @@ private Mono<Void> processNewLocationUrl(ServerWebExchange exchange, Config conf
 
         if (newUrl.get() != null) {
             // if the new URL was defined, decorate (scheme by AT-TLS) and set
-            if (isServerAttlsEnabled) {
+            if (isServerAttlsEnabled && newUrl.get().startsWith("http")) {
```

Comment:
```suggestion
            if (isServerAttlsEnabled && newUrl.get().contains("//")) {
```

This would be activated also in case of relative path to `http*` endpoints, e.g. redirect in actuator to httptrace.

Reply:
we agreed to keep the schema relative path. What has changed?